ui/tests/acceptance/sync/secrets/overview-test.js

```diff
@@ -140,77 +139,76 @@ module('Acceptance | sync | overview', function (hooks) {
         'create new destination is available once feature is activated'
       );
     });
-  });
 
-  module('enterprise with namespaces', function (hooks) {
-    hooks.beforeEach(async function () {
-      this.version.features = ['Secrets Sync', 'Namespaces'];
-      await runCmd(`write sys/namespaces/admin -f`, false);
-      await authPage.loginNs('admin');
-      await runCmd(`write sys/namespaces/foo -f`, false);
-      await authPage.loginNs('admin/foo');
-    });
+    module('enterprise with namespaces', function (hooks) {
+      hooks.beforeEach(async function () {
+        this.version.features = ['Secrets Sync', 'Namespaces'];
 
-    test('it should make activation-flag requests to correct namespace', async function (assert) {
-      assert.expect(4);
-      // should call GET activation-flags twice because we need an updated response after activating the feature
-      this.server.get('/sys/activation-flags', (_, req) => {
-        assert.deepEqual(req.requestHeaders, {}, 'Request is unauthenticated and in root namespace');
-        return {
-          data: {
-            activated: [''],
-            unactivated: ['secrets-sync'],
-          },
-        };
-      });
-      this.server.post('/sys/activation-flags/secrets-sync/activate', (_, req) => {
-        assert.strictEqual(
-          req.requestHeaders['X-Vault-Namespace'],
-          undefined,
-          'Request is made to undefined namespace'
-        );
-        return {};
+        await runCmd(`write sys/namespaces/admin -f`, false);
+        await authPage.loginNs('admin');
+        await runCmd(`write sys/namespaces/foo -f`, false);
+        await authPage.loginNs('admin/foo');
```

Comment:
Is the full test suite passing locally? Typically with namespace tests we have to manually clear the auth form namespace query params otherwise they stick for subsequent test runs. 

https://github.com/hashicorp/vault/blob/main/ui/tests/acceptance/secrets/backend/kv/kv-v2-workflow-edge-cases-test.js#L422